test/varity/hgvs_to_vcf_test.clj

```diff
@@ -14,6 +14,36 @@
                                      test-ref-seq-file]]
             [varity.vcf-to-hgvs :as v2h]))
 
+(deftest ->supported-transcript-test
```

Comment:
Is it ok to add private function's test?

Reply:
Of course! 👍